data/lua/on_event.lua

```diff
@@ -2,54 +2,33 @@
 -- so you have to call this function from a toplevel lua tag or from a preload event.
 -- It is also not possible to use this for first_time_only=yes events.
 
-if rawget(_G, "core_on_event") then
-	return rawget(_G, "core_on_event")  -- prevent double execution
-end
-
-local event_handlers = {}
-
-local old_on_event = wesnoth.game_events.on_event or function(eventname) end
-wesnoth.game_events.on_event = function(eventname)
-	old_on_event(eventname)
-	local context = nil
-	for _, entry in pairs(event_handlers[eventname] or {}) do
-		if context == nil then
-			context = wesnoth.current.event_context
-		end
-		entry.h(context)
-	end
-end
+-- This api used to be the default way to add events from lua (with its own implementation in lua
+-- based on game_events.on_event). Now its just calls game_evets.add, in particular because otherwise the
+-- priority parammter wouldn't work accord the differnt implmentaions..
+-- Still kept now for compatibility and because it has an easier to
+-- use interface meaning you can easily write in a lua file:
+--
+-- on_event("moveto", 10, function(ec)
+--   ...
+-- end)
+--
+-- which is imo more convenient than the interace wesnoth.game_events.add or wesnoth.game_events.add_repeating offers
+-- even though its at this point technicially equivalent to the later.
```

Comment:
Typo: "technically" has only one i

Reply:
Also, that last word should be "latter" with two T's. And you misspelled "interface" on the line above.